benchmarks/DL3/benchmarks_DL3_IRFs_and_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,923 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# IRFs and sensitivity pyirf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**WARNING**\n",
+    "\n",
+    "This is still a work-in-progress, it will evolve with the pipeline comparisons and converge with ctaplot+cta-benchmarks.\n",
+    "\n",
+    "**IMPORTANT**\n",
+    "\n",
+    "Soon this will be supersided by new results using [pyirf](https://github.com/cta-observatory/pyirf) and ctaplot metrics."
```

Comment:
You can remove this section since now we are using pyirf and we mention ctaplot above